composer.json

```diff
@@ -47,7 +47,8 @@
   "extra": {
     "google/apiclient-services": [
       "Calendar",
-      "Drive"
+      "Drive",
+      "Directory"
```

Comment:
Old calendar ID also does not have Directory permission then how come attendees are showing in that?